example-testng-fork-execution/suites/logging_tests.xml

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="Logging Tests" parallel="classes" thread-count="10">
```

Comment:
Do we really need to additionally check that in a multi-thread execution?

Reply:
removed